README.md

```diff
@@ -40,18 +31,14 @@ class Header extends Component {
     render() {
         return (
             <div>
-                <EvergageAB campaign="logoTest" variants={[
-                    {
-                        name: "headingTest",
-                        node: <h1>Test header</h1>
-                    }, {
-                        name: "spanHeading",
+                <EvergageAB campaign="logoTest" variants={[{
                         node: <span>Test header in span</span>
```

Comment:
if we are removing the name property from a variant, is it necessary to make this an array of objects? we could just an array of JSX